doc/tutorials/using_celadon_as_uos.rst

```diff
@@ -0,0 +1,113 @@
+.. _using_celadon_as_uos:
+
+Using Celadon as User OS
+########################
+
+`Celadon <https://01.org/projectceladon/>`_ is an open source Android* software reference stack
+for Intel architecture. It builds upon a vanilla Android stack and incorporates open sourced components
+that are optimized for the hardware. This tutorial describes how to run Celadon as the User OS
+on the ACRN hypervisor. We are using Kaby Lake-based NUC (model NUC7i7DNHE) in this tutorial.
+Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to OVMF
+********************
+
+Open Virtual Machine Firmware (OVMF) is an `EDK II
+<https://github.com/tianocore/tianocore.github.io/wiki/EDK-II>`_
+based project to enable `UEFI <https://github.com/tianocore/tianocore.github.io/wiki/UEFI>`_
+support for Virtual Machines. OVMF contains sample UEFI firmware for `QEMU <http://www.qemu-project.org/>`_ and `KVM
+<http://www.linux-kvm.org/>`_.
+
+Prerequisites
+*************
+
+* Ubuntu 16.04 or higher with at least 150G free disk space
+* Intel NUC Kit ( Reference Platforms: :ref:`ACRN supported platforms <hardware>` )
```

Comment:
Yes, I've checked ``device mode`` option is also exit in the APL NUC Bios setting.

Reply:
@lirui34 It looks like the USB Device mode is supported since firmware 0059 for NUC6CAYH, and firmware NUC7ixDNH. Suggest to call call it out explicitly. :)

Reply:
Thanks for the information, I will suggest people to use 0059 or above so that they will also see the ``device mode`` in APL BIOS settings.
Unfortunately in my latest test result based on NUC7i7BNH and NUC6CAYH, there's no Celadon graphic on those platforms with only one HDMI port. Since I am not sure it's a Celadon issue or ACRN issue, so I don't want to mention the APL in this introduction.